.travis.yml

```
@@ -4,9 +4,9 @@ go_import_path: go.uber.org/dig
 
 matrix:
   include:
-    - go: 1.7
-    - go: 1.8
     - go: 1.9
+    - go: 1.10
```

```suggestion
    - go: "1.10"
```

need quotes because "1.10" as float is "1.1"